packages/fxa-auth-server/lib/routes/account.js

```diff
@@ -685,6 +689,26 @@ module.exports = (
           sessionToken = await db.createSessionToken(sessionTokenOptions);
         }
 
+        function forcePasswordChange(account) {
+          // If it's an email address used for testing etc,
+          // we should force password change.
+          if (
+            config.forcePasswordChange &&
+            config.forcePasswordChange.forcedEmailAddresses
+          ) {
+            if (
+              config.forcePasswordChange.forcedEmailAddresses.test(
+                account.primaryEmail.email
+              )
+            ) {
+              return true;
+            }
```

Comment:
I don't see a reason for the inner `if`.  Can it be the third comparison in the outer `if`?